bundles/org.dataflowanalysis.analysis.dfd/src/org/dataflowanalysis/analysis/dfd/core/DFDTransposeFlowGraphFinder.java

```diff
@@ -17,7 +17,7 @@
  */
 public class DFDTransposeFlowGraphFinder implements TransposeFlowGraphFinder {
     private final DataDictionary dataDictionary;
-    private final DataFlowDiagram dataFlowDiagram;
+    protected final DataFlowDiagram dataFlowDiagram;
```

Comment:
Why do you need protected in this class?

Reply:
because i need access to the dataFlowDiagram in the child
Could introduce getter

Reply:
I talked with sebastian, protected should be fine